concertfit/Pages/TicketMaster.cshtml.cs

```diff
@@ -12,31 +12,18 @@ public class TicketMasterModel : PageModel
     {
         public void OnGet()
         {
-            String myName = "Shubham Das";
-            int age = 25;
-            ViewData["MyName"] = myName;
-            ViewData["age"] = age;
-
             using (WebClient webClient = new WebClient())
             {
                 // get the raw JSON data.
                 string jsonDataTicketmaster = webClient.DownloadString("https://concertfitapis20191118021024.azurewebsites.net/api/ticketmaster");
                 TicketMasterResponse.TopLevel topLevelTicketMaster = TicketMasterResponse.TopLevel.FromJson(jsonDataTicketmaster);
+                ViewData["TicketMaster"] = topLevelTicketMaster;
 
                 string jsonDataSpotify = webClient.DownloadString("https://concertfitapis20191118021024.azurewebsites.net/api/spotify");
                 SpotifyArtistsResponse.TopLevel[] topLevelSpotify = SpotifyArtistsResponse.TopLevel.FromJson(jsonDataSpotify);
+                ViewData["SpotifyMaster"] = topLevelSpotify;
+
 
-                //Console.WriteLine(topLevel.ToString());
```

Comment:
Delete this.  It's a no-no to commit commented code to VCS.  You can always go back in history to see what used to be there.